com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/ApplicationResources.java

```diff
@@ -0,0 +1,28 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.util.Collection;
+
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.domain.CloudResource;
+import org.cloudfoundry.client.lib.domain.CloudResources;
+
+public class ApplicationResources {
+    private CloudResources cloudResources;
+
+    public void createCloudResources(Collection<CloudResource> cloudResourceCollection) {
+        this.cloudResources = new CloudResources(cloudResourceCollection);
+    }
+
+    public CloudResources getCloudResources() {
+        if (cloudResources == null) {
```

Comment:
If the above is a constructor this will not be needed.

Reply:
fixed